src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskProviderTest.java

```diff
@@ -126,6 +124,17 @@ private void setInitMockTaskWithWarnings() {
     }
```

Comment:
the following seem are deprecated in version 4.3.4 ->
 scheme = "${PACT_BROKER_SCHEME:http}",
    host = "${PACT_BROKER_URL:localhost}",
    port = "${PACT_BROKER_PORT:9292}",

Can we try using the "url" as suggested in the doc on the pr for one providers and see if it works on our pact broker?